pkg/pillar/dpcreconciler/linux.go

```diff
@@ -1333,11 +1349,14 @@ func (r *LinuxDpcReconciler) getWifiCredentials(wifi types.WifiConfig) (types.En
 }
 
 func (r *LinuxDpcReconciler) getIntendedWwanConfig(dpc types.DevicePortConfig,
-	aa types.AssignableAdapters, radioSilence bool) dg.Item {
+	aa types.AssignableAdapters, radioSilence bool, gcp types.ConfigItemValueMap) dg.Item {
+	logLevel := gcp.AgentSettingStringValue("wwan", types.LogLevel)
 	config := types.WwanConfig{
 		RadioSilence: radioSilence,
+		Verbose:      logLevel == "debug" || logLevel == "trace",
```

Comment:
Does logrus package have defined values for the log levels which we can use here? Something that would enable saying loglevel <= debug based on parsed string to enum?